.pipelines/e2e-windows.yaml

```diff
@@ -72,4 +72,32 @@ stages:
       windowsVMSize: Standard_DS2_v2
       windowsDistro: aks-windows-2022-containerd-gen2
       windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
+      storageAccount: akswinstore2022gen2
+
+- stage: test_2022_containerd_gen2_gpu_grid
+  dependsOn: []
+  condition: eq('${{ parameters.test2022containerdgen2 }}', True)
+  jobs:
+  - template: ./templates/e2e-windows-template.yaml
+    parameters:
+      windowsImage: 2022-containerd-gen2
+      windowsOSSKU: Windows2022
+      windowsVMSize: Standard_NV6
+      windowsDistro: aks-windows-2022-containerd-gen2
+      windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
+      windowsGpuDriverSuffix: -grid
+      storageAccount: akswinstore2022gen2
+
+- stage: test_2022_containerd_gen2_gpu_cuda
+  dependsOn: []
+  condition: eq('${{ parameters.test2022containerdgen2 }}', True)
+  jobs:
+  - template: ./templates/e2e-windows-template.yaml
+    parameters:
+      windowsImage: 2022-containerd-gen2
+      windowsOSSKU: Windows2022
+      windowsVMSize: Standard_nc6_promo
```

Comment:
Standard_NC6_prom is V1 generation vm_size, it cannot be used together with gen2 distro aks-windows-2022-containerd-gen2

Reply:
use Standard_NC6s_v3 instead